packages/next/client/trusted-types.ts

```diff
@@ -35,3 +63,14 @@ export function __unsafeCreateTrustedScriptURL(
 ): TrustedScriptURL | string {
   return getPolicy()?.createScriptURL(url) || url
 }
+
+/**
+ * Returns true if the input is a TrustedScript object, and returns false
+ * otherwise.
+ */
+export function isTrustedScript(script: any): boolean {
+  return (
+    (typeof window !== 'undefined' && window.trustedTypes?.isScript(script)) ||
+    false
```

Comment:
Did you forget to drop `|| false` here?

Reply:
`!!` is now used instead to improve readability.